azure-pipelines.yml

```diff
@@ -33,13 +33,24 @@ stages:
       inputs:
```

Comment:
its identical to this https://github.com/OfqualGovUK/ofqual-register-api/pull/24